Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -208,14 +208,14 @@ def create_ticket_context(data: dict, additional_fields: list = None) -> Any:
 
     # These fields refer to records in the database, the value is their system ID.
     closed_by = data.get('closed_by')
-    if closed_by:
+    if closed_by and type(closed_by) == dict:
         context['ResolvedBy'] = closed_by.get('value', '')
     opened_by = data.get('opened_by')
-    if opened_by:
+    if opened_by and type(opened_by) == dict:
```

Comment:
What about all the else cases here?

Reply:
I fixed it
I attached 2 pictures to understand the difference when using the ***sysparm_exclude_reference_link=true*** parameter
![Screen Shot 2021-05-18 at 17 24 09](https://user-images.githubusercontent.com/78307768/118669653-9851b580-b7fe-11eb-9034-7ba55b470647.png)
![Screen Shot 2021-05-18 at 17 25 14](https://user-images.githubusercontent.com/78307768/118669668-9a1b7900-b7fe-11eb-8733-0c57dada86da.png)